tests.toml

```
@@ -0,0 +1,15 @@
+test_format = 1.0
+
+[default]
+
+    # ------------
+    # Tests to run
+    # ------------
+
+    exclude = ["install.private", "install.root", "install.subdir", "install.multi"]
```

```suggestion
    exclude = ["install.private"]
```

not sure why we should be excluding those, it's a webapp (though mainly technical but it does have an nginx conf) and multi_instance is enabled ... `install.subdir` is already excluded because there's no `path` question ... i guess we can exclude `install.private` because it's not that much important